### PR TITLE
add documentation for ModelInitializerBase in eden/model_base.py

### PR DESCRIPTION
**Pull request by tzk**
This would probably be useful to implement a model driver without reading the other 400 lines of code. What do you thinkg. This draft is incomplete and possibly wrong.

### CODE REVIEW

File: eden/model_base.py [hunk 1]
```diff
@@ -32,34 +32,69 @@
...
 
     def load_positive_data(self, args):
+        """Load the positive class data.
+        
+        Called after invocating the fit command.
+        
+        The iterator returned here is handled by the pre_processor function
+        defined in pre_processor_init.
+        """
+        return self.load_data(args.positive_input_file)
```
Comment (bgruening) [inline eden/model_base.py]: You added here a return, but did not remove the two lines below.

Comment (tzk) [inline eden/model_base.py]: thanks for spotting, removed the superfluous line

**Comment by bgruening** [comment]
@tzk this LGTM. I'm not sure if you should add TODO to the documentation or simply leave this undocumented. I will leave this to be merge by @fabriziocosta. He should decide :)

**Comment by tzk** [comment]
well, @fabriziocosta knows what's right and what's wrong. please fix or tell :)

**Pull request merged**